Swiften/Elements/ClientState.h

```diff
@@ -18,14 +18,18 @@ namespace Swift {
 
         public:
             enum ClientStateType {Active, Inactive};
-            ClientState(ClientStateType state = Active) {
+            ClientState() {
+
+            }
+
+            ClientState(ClientStateType state) {
                 state_ = state;
```

Comment:
The `state_` member should be initialised in the initialiser list. Like previously suggested ` ClientState(ClientStateType state) : state_(state) {`.